### PR TITLE
NWPS-913: Upgrade cloud domain from '${env}-{brcStack}.onehippo.io' to '${env}.{brcStack}.bloomreach.cloud'

### PR DESCRIPTION
**Pull request by smanivasag**
Note that a separate PR would be created to update Github Action versions once the following PRs have been reviewed and merged:
- https://github.com/Health-Education-England/cleanup-distributions-from-BR-Cloud-action/pull/11
- https://github.com/Health-Education-England/upload-distribution-to-BR-Cloud-action/pull/6
- https://github.com/Health-Education-England/deploy-distribution-to-BR-Cloud-action/pull/5

### CODE REVIEW

**Comment by smanivasag** [comment]
Hi @sankar-shunmuga - Have made a commit b1f11ab57bec1055358ec8a022dc98f30eef470b containing CI/CD changes to use the latest versions of Github Actions (which in turn contains updates for `api.${brcStack}.bloomreach.cloud` upgrade). Could you review ?

**Comment by smanivasag** [comment]
Hi @sankar-shunmuga  @FranciscoRuizHEE - Have made another commit 09b9f95f3507e67a95722bded5a2f30a8b2f0f43 containing the following changes:
- Put back the virtual host support for current `{env}-{namespace}.onehippo.io` domains as agreed until we roll out the support for `{env}.{namespace}.bloomreach.cloud` domains onto prod and verified.
- Replacing env specific `{env}.{namespace}.bloomreach.cloud` virtual hosts with generic virtual hosts (`/hst:platform/hst:hosts/br-cloud` & `/hst:hee/hst:hosts/br-cloud`) using [System Property Expansion in Virtual Host Configuration](https://xmdocumentation.bloomreach.com/library/enterprise/installation-and-configuration/configure-virtual-hosts-in-an-environment.html#scroll_use-system-property-expansion-in-virtual-host-configuration). Note that this change will add additional authentication for non-prod pages due to `hst:authenticated=true` & `hst:roles=site.viewer` properties on `/hst:hee/hst:hosts/br-cloud/cloud/bloomreach/${brc.namespace}/${brc.environmentname}/hst:root`, but I think this is Ok. Let me know your thoughts.

The PR is now ready for review again. Thanks!

File: repository-data/application/src/main/resources/hcm-config/hst/hosts/br-cloud/cloud.yaml [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+        /${brc.namespace}:
           jcr:primaryType: hst:virtualhost
-          /cms-prod:
+          /${brc.environmentname}:
             jcr:primaryType: hst:virtualhost
             /hst:root:
               jcr:primaryType: hst:mount
               hst:ismapped: false
               hst:namedpipeline: WebApplicationInvokingPipeline
-              hst:alias: cms-mount
```
Comment (sankar-shunmuga) [inline repository-data/application/src/main/resources/hcm-config/hst/hosts/br-cloud/cloud.yaml:18]: I think we need this alias as it has been referred in BlogPostCommentMailFormDataBehavior.java

**Review by sankar-shunmuga — COMMENTED** [review]
Thanks @smanivasag . it looks fine,
Can you please clarify the comment ?

**Comment by smanivasag** [comment]
Thanks @sankar-shunmuga, good catch. Added `cms-mount` alias back to the generic platform vhost `br-cloud`.

**Review by sankar-shunmuga — APPROVED** [review]

**Pull request merged**